### PR TITLE
add tasks for permission to ssl

### PR DESCRIPTION
**Pull request by Xolog**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@
 fluentd_migrate_to_v5: false
 fluentd_del_old_conf: false
 
+# for ssl 
+ssl_directory_path: "/etc/ssl_certs"
+domen_directory: "example.com"
```
Comment (sorrowless) [inline defaults/main.yml]: domain

File: tasks/install.yml [hunk 2]
```diff
@@ -47,3 +47,28 @@
     group: "fluentd"
     mode: 0755
     state: directory
+
+- name: Fluentd | Check if the directory for ssl exists
+  stat:
+    path: "{{ ssl_directory_path }}/{{ domen_directory }}"
```
Comment (sorrowless) [inline tasks/install.yml]: domain

File: tasks/install.yml [hunk 3]
```diff
@@ -47,3 +47,28 @@
...
+  register: dir_stat
+
+- name: Fluentd | Find certs in the directory
+  find:
+    paths: "{{ ssl_directory_path }}/{{ domen_directory }}"
```
Comment (sorrowless) [inline tasks/install.yml]: domain

File: tasks/install.yml [hunk 4]
```diff
@@ -47,3 +47,28 @@
...
+- name: Fluentd | Set 644 permissions for certs
+  file:
+    path: "{{ item.path }}"
+    mode: '0644'
+  loop: "{{ cert_files.files }}"
+  when: dir_stat.stat.exists
+
+- name: Fluentd | Set 755 permissions for the directory
+  file:
+    path: "{{ ssl_directory_path }}/{{ domen_directory }}"
```
Comment (sorrowless) [inline tasks/install.yml]: domain

File: defaults/main.yml (hunk 1 above)
Comment (Xolog) [reply defaults/main.yml]: fix

File: tasks/install.yml (hunk 2 above)
Comment (Xolog) [reply tasks/install.yml]: fix

File: tasks/install.yml (hunk 4 above)
Comment (Xolog) [reply tasks/install.yml]: fix

File: tasks/install.yml (hunk 3 above)
Comment (Xolog) [reply tasks/install.yml]: fix

File: README.md [hunk 5]
```diff
@@ -10,13 +10,21 @@ Ansible 2.4
 #### Role Variables
 
 ```yaml
+# true if td-agent v4 is installed on the host
+fluentd_migrate_to_v5: false
+fluentd_del_old_conf: false
+
+# for ssl 
+ssl_directory_path: "/etc/ssl_certs"
+domen_directory: "example.com"
```
Comment (sorrowless) [inline README.md]: domain

Comment (Xolog) [reply README.md]: fix

**Review by sorrowless — APPROVED** [review]

**Pull request merged**